src/SfxWeb/src/app/views/node-type/node-types/node-types.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, Injector, OnInit } from '@angular/core';
+import { map } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+import { IResponseMessageHandler } from 'src/app/Common/ResponseMessageHandlers';
+import { NodeCollection } from 'src/app/Models/DataModels/collections/NodeCollection';
+import { ListSettings, ListColumnSettingForLink, ListColumnSetting, ListColumnSettingWithFilter, ListColumnSettingForBadge } from 'src/app/Models/ListSettings';
+import { DataService } from 'src/app/services/data.service';
+import { SettingsService } from 'src/app/services/settings.service';
+import { IDashboardViewModel, DashboardViewModel } from 'src/app/ViewModels/DashboardViewModels';
+import { NodeTypeBaseControllerDirective } from '../NodeTypeBase';
+
+@Component({
+  selector: 'app-node-types',
+  templateUrl: './node-types.component.html',
+  styleUrls: ['./node-types.component.scss']
+})
+export class NodeTypesComponent extends NodeTypeBaseControllerDirective {
+
+  nodes: NodeCollection;
+  listSettings: ListSettings;
+  tiles: IDashboardViewModel[] = [];
+  typeCollection: any[];
+
+  constructor(protected data: DataService, private settings: SettingsService, injector: Injector) {
+    super(data, injector);
+   }
+
+  setup() {
+    this.nodes = this.data.nodes;
+    this.listSettings = this.settings.getNewOrExistingListSettings('nodes', ['name'], [
```

Comment:
move this to the settings service to reuse it and dont include the node type for the node type page.